dace/frontend/python/newast.py

```diff
@@ -375,18 +396,20 @@ def add_indirection_subgraph(sdfg: SDFG,
                 # TODO: Make this work for nested for-loops
                 arr_rng = dace.subsets.Range([(a, a, 1) for a in access])
                 if output:
-                    arrname, _ = PVisitor._add_write_access(arr_name,
+                    arrname, rng = PVisitor._add_write_access(arr_name,
                                                             arr_rng,
                                                             target=None)
                 else:
-                    arrname, _ = PVisitor._add_read_access(arr_name,
+                    arrname, rng = PVisitor._add_read_access(arr_name,
```

Comment:
are you supposed to use `rng` somewhere? There's `arr_rng` below and the subset range that we create from the whole array

Reply:
I was going to use it, but I found `subsets::Range::from_array` better suited (for the time being).